### DISCUSSION TITLE
Performing bulk partial update

### DISCUSSION THREAD

**Discussion by rnv812** [encode/django-rest-framework#8935] [Question & Answer]
Default `UpdateModelMixin` works on single instances. Is there any simple way to perform bulk update not implementing it by myself?

**Comment by rnv812** [comment]
Were any new features added since this discussion #7851 ?

**Reply by auvipy** [reply]
I think they followed a workaround https://github.com/encode/django-rest-framework/discussions/7851#discussioncomment-4771702
